packages/flutter_form_bloc/lib/src/date_time/date_time_field_bloc_builder_base.dart

```diff
@@ -235,16 +235,22 @@ class _DateTimeFieldBlocBuilderBaseState<T>
   }
 
   Future<TimeOfDay> _showTimePicker(BuildContext context) async {
```

Comment:
To reduce conditions imbrication, you could early exit style :

```dart
class _DateTimeFieldBlocBuilderBaseState<T> ... {

  TimeOfDay _initialTime() {
    if (widget.dateTimeFieldBloc.state.value == null) {
      return widget.initialTime;
    }
    if (widget.type == DateTimeFieldBlocBuilderBaseType.time) {
      return widget.dateTimeFieldBloc.state.value;
     }
     return TimeOfDay.fromDateTime(widget.dateTimeFieldBloc.state.value);
  }

  Future<TimeOfDay> _showTimePicker(BuildContext context) async {
    return await showTimePicker(
      context: context,
      useRootNavigator: widget.useRootNavigator,
      initialTime: _initialTime(),
      builder: widget.builder,
    );
  }
}
```

Reply:
It seems that no one cares about this project anymore(